cryptotest/tests/KeyPairGeneratorTests.java

```diff
@@ -80,6 +80,14 @@ protected void checkAlgorithm(Provider.Service service, String alias) throws
                 keySize = 2048;
             } else if (service.getAlgorithm().contains("RSA")) {
                 keySize = 2048;
+            } else if (service.getAlgorithm().contains("ML-")) {
+                // keySize is intentionally -1 here, KPG of this provider [1]
+                // does not override default initialize method [2],
+                // internal (in-tree) tests do the same [3], see:
+                // [1] https://github.com/openjdk/jdk/blob/da2b4f0749dffc99fa42c7311fbc74231af273bd/src/java.base/share/classes/com/sun/crypto/provider/ML_KEM_Impls.java#L40
+                // [2] https://github.com/openjdk/jdk/blob/da2b4f0749dffc99fa42c7311fbc74231af273bd/src/java.base/share/classes/sun/security/provider/NamedKeyPairGenerator.java#L153
+                // [3] https://github.com/openjdk/jdk/blob/da2b4f0749dffc99fa42c7311fbc74231af273bd/test/jdk/sun/security/provider/all/Deterministic.java#L208
+                keySize = -1;
```

Comment:
thats weird

Reply:
Probably not supposed to specify key for these. But as in-tree jtregs do the same I assume this is by design.

Reply:
Yah, I was reading the links you posted. Thanx a lot for them! Wicked -1, one would say we are far from that already...

Reply:
:)